modules/adxpremiumAnalyticsAdapter.js

```diff
@@ -19,18 +21,26 @@ const {
   }
 } = CONSTANTS;
 
+let timeoutBased = false;
+let requestSent = false;
```

Comment:
`requestSent` (probably `requestDelivered` also) are "global variables" for this adapter. From what I see this variable is set to `true` by `clearSlot()`, which is triggered by gpt's slotRenderEnded event. As this variable is global, that means, when a publisher is running multiple slots on the page, slotRenderEnded of one slot would set this variable to `true` and `sendEvent(completeObject);` would only trigger for that one slot, skipping all the rest. Am I missing something, or is this intentional?

Reply:
This is used just for the timing purposes of the send event and it becomes false again when the new auctionInit event occurs. I will take another look at it, for now it works fine.